desktop-src/sr/calling-srsetrestorepoint.md

```diff
@@ -1,36 +1,32 @@
+- If the key does not exist (default) and any restore points have been created in the last 24 hours, Windows skips creating this new restore point. System Restore sets the `IISequenceNumber` member of the [`STATEMGRSTATUS`](/windows/win32/api/srrestoreptapi/ns-srrestoreptapi-statemgrstatus) structure to the sequence number for the restore point created previously in the day and sets the value of the `nStatus` member to `ERROR_SUCCESS`. The **SRSetRestorePoint** function returns `TRUE`.
 
-Developers can write applications that create the **DWORD** value **SystemRestorePointCreationFrequency** under the registry key **HKLM\\Software\\Microsoft\\Windows NT\\CurrentVersion\\SystemRestore**. The value of this registry key can change the frequency of restore point creation. The value of this registry key can change the frequency of restore point creation.
+- If the registry key value is 0, system restore does not skip creating the new restore point.
 
-If the application calls **SRSetRestorePoint** to create a restore point, and the registry key value is 0, system restore does not skip creating the new restore point.
-
-If the application calls **SRSetRestorePoint** to create a restore point, and the registry key value is the integer N, system restore skips creating a new restore point if any restore points were created in the previous N minutes.
-
-**Windows 8:  **
+- If the registry key value is the integer N, system restore skips creating a new restore point if any restore points were created in the previous N minutes.
 
 System Restore running on Windows 8 monitors files in the boot volume that are relevant for system restore only. Snapshots of the boot volume created by System Restore running on Windows 8 may be deleted if the snapshot is subsequently exposed by an earlier version of Windows. Note that although there is only one system volume, there is one boot volume for each operating system in a multi-boot system.
 
-Developers can write applications that create the **DWORD** value **ScopeSnapshots** under the registry key **HKLM\\Software\\Microsoft\\Windows NT\\CurrentVersion\\SystemRestore**. If this registry key value is 0, System Restore creates snapshots of the boot volume in the same way as in earlier versions of Windows. If this value is deleted, System Restore running on Windows 8 resumes creating snapshots that monitor files in the boot volume that are relevant for system restore only.
+Developers can write applications that create the DWORD value **ScopeSnapshots** under `HKLM\Software\Microsoft\Windows NT\CurrentVersion\SystemRestore` the registry key. If this registry key value is 0, System Restore creates snapshots of the boot volume in the same way as in earlier versions of Windows. If this value is deleted, System Restore running on Windows 8 resumes creating snapshots that monitor files in the boot volume that are relevant for system restore only.
```

Comment:
One minor suggestion. The rest of this PR looks great. Thank you for the comprehensive update.
```suggestion
Developers can write applications that create the DWORD value **ScopeSnapshots** under the `HKLM\Software\Microsoft\Windows NT\CurrentVersion\SystemRestore` registry key. If this registry key value is 0, System Restore creates snapshots of the boot volume in the same way as in earlier versions of Windows. If this value is deleted, System Restore running on Windows 8 resumes creating snapshots that monitor files in the boot volume that are relevant for system restore only.
```